catalog/.gitignore

```diff
@@ -1,3 +1,7 @@
+node_modules
+ipynb-styling/notebook
```

Comment:
I don't have this directory, is it your personal stuff?

Reply:
these directories are created when building ipynb styles (see readme and makefile in ipynb-styling)

Reply:
:ok_hand: